### DISCUSSION TITLE
Aligning line plots and information on second y-axis

### DISCUSSION THREAD

**Discussion by amegbor** [teunbrand/ggh4x#142] [Q&A] [answered]
Hello everyone,

I'm seeking assistance in aligning my coefficient plots with the text information displayed on the second y-axis on the right. I've attempted to achieve this using the code below, and here's the current result. I also experimented with manipulating the values in breaks = as.vector(outer(c())), but it seems to worsen the alignment of the plot. Any help or suggestions will be greatly appreciated. I can provide the data if needed. Thank you.

`ggplot(df, aes(x=mean, y = Exposure, 
                            colour=Models)) + 
  scale_color_brewer(palette="Set1",
                     breaks=c("Model 1","Model 2","Model 3", "Model 4","Model 5")) + 
  geom_vline(aes(xintercept = 0), size = .25, linetype = "dashed") + 
  geom_point(position = position_dodge(width=.75)) + 
  scale_x_continuous(breaks = c(-0.1, 0.001, 0.03, 0.05, 0.08)) + 
  geom_errorbarh(aes(xmin = lci, xmax=uci), position=position_dodge(width=.75), height=0) + 
  labs(x="Posterior Mean", y="Exposure", colour="Models") +
  guides(
    y = guide_axis_nested(delim = "&", n.dodge = 1),
    y.sec = guide_axis_manual(
      breaks = as.vector(outer(c(-0.18125, -0.09375, 0.09375, 0.18125, 0.36875), 1:2, "+")),
      labels = rev(df$estimates)
    )
  ) +
  theme_classic() +
  theme(
    axis.text.y.left = element_text(margin = margin(r = 5, l = 5)),
    ggh4x.axis.nesttext.y = element_text(margin = margin(r = 6, l = 6)),
    ggh4x.axis.nestline.y = element_blank())
`

![Result_fixed](https://github.com/teunbrand/ggh4x/assets/60387204/c6517dca-c886-4701-9692-bf55f96a3de8)

**Comment by teunbrand** [comment, marked as answer]
I think it might be a matter of just choosing the appropriate values.
You have `width = 0.75` with `n = 5` groups, so each group should be spaced `0.75 / 5 == 0.15` y-axis units apart.
So the values for the manual axis should be:
```r
as.vector(outer(c(-0.3, -0.15, 0, 0.15, 0.3), 1:2, "+"))
```

**Reply by amegbor** [reply]
Thank you. it worked.
